doc/rst/source/clip.rst

```diff
@@ -41,18 +42,23 @@ Synopsis
 Examples
 --------
 
-To set up a complex clip
-area to which all subsequent plotting will be confined, run:
+To see the effect of a simple clip path which result in some symbols
+being partly visible or missing altogether, try
 
    ::
 
-    gmt clip my_region.xy -R0/40/0/40 -Jm0.3i
-
-To deactivate the clipping in an existing plotfile, run:
-
-   ::
-
-    gmt clip -C
+    gmt begin clip
+      gmt clip -R0/6/0/6 -Jx2.5c -W1p,blue << EOF
+0 0
```

Comment:
I think the indentations here are incorrect.

Reply:
You mean you want space in front of those coordinates?

Reply:
Yes, the coordinates should have the same indentations as `gmt begin`. 

This is now what it looks like:
![image](https://user-images.githubusercontent.com/3974108/65381424-ec6d6500-dcbe-11e9-9a75-fbfe863866ad.png)